pkg/controller/installation/validation.go

```diff
@@ -349,6 +349,13 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 			}
 
 		}
+
+		if instance.Spec.CalicoNetwork.PolicySetupTimeoutSeconds != nil {
```

Comment:
I think we need validation here to prevent setting this in other invalid scenarios - e.g., when Calico networking isn't being used, or when Calico in eBPF mode is being used.

Reply:
What about Windows? Do we prevent this option in a cluster with Windows nodes? Do we just ignore it?

Reply:
I've been working off the assumption that we would ignore this option on Windows, how does the operator deal with the issue of windows compat? Does it know if a windows node exists in the cluster?

Reply:
I suppose I could accept it in the operator, ignore it in felix/cni, if thats a worthy paradigm?

Reply:
Typically, we would expect that if a config option isn't supported in a certain configuration, that we return an error and prevent installation of the code in that configuration. We should probably do that for this if Windows is enabled in the configuration.

Reply:
Im unfamiliar with this part of the operator resources, but my first thought was to check if the windows dataplane config option was non-nil. But, that would presumably only tell me that _some_ nodes may be windows, and I still may want the feature on for non-windows nodes :thinking: any guidance appreciated

Reply:
There's no such thing as a "Windows only" cluster - they all have at least one Linux node since the control plane components don't run on Windows.

The usual ideal we strive for is that if configuration is accepted it means it is going to be implemented fully. We don't really have a mechanism to report back "This configuration is sort of working but not entirely"

I think your options are:
- Reject the config outright if Windows is enabled.
- Add support for this feature to Windows.